sdk/keyvault/azure-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/HelloWorld.java

```diff
@@ -36,14 +36,14 @@ public static void main(String[] args) throws InterruptedException, IllegalArgum
             .expires(OffsetDateTime.now().plusYears(1)));
```

Comment:
It would be nice to have a couple of samples showing usage of Response <T> in Hello World area.
Processing the headers info.
Doesn't have to be everywhere.

Reply:
KeyVault API responses don't seem to have a lot of information on the headers that could be useful for showing in samples.
I just added something that could validate the operation using the status code information in 
the response.

Reply:
Yeah that is fine for now, the goal is to show the use of Response<T>
extracting key , and looking at whatever header info is given.